code/modules/mob/language.dm

```diff
@@ -486,6 +486,14 @@
 	flags = RESTRICTED | HIVEMIND | NOBABEL
 	follow = TRUE
 
+/datum/language/terrorspider/broadcast(mob/living/speaker, message, speaker_mask)
+	if(isterrorspider(speaker))
+		var/mob/living/simple_animal/hostile/poison/terror_spider/T = speaker
+		if(T.spider_tier >= 3 || istype(T, /mob/living/simple_animal/hostile/poison/terror_spider/white)) //I think whites are important enough for this, but it can be changed. Defines not global, 3 must be used.
```

Comment:
Snowflakey, you should define a boolean at the terror spider level to decide whether to amplify their speech

Reply:
Done!